### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a few typos I spotted.

### CODE REVIEW

File: diagrams.net/assets/stencils/aws/deployment_management.xml [hunk 1]
```diff
@@ -253,7 +253,7 @@
 <fillstroke/>
 </foreground>
 </shape>
-<shape name="AWS Elastic BeanStalk Applicaton" h="53.89" w="31.1" aspect="variable" strokewidth="inherit">
```
Comment (ggrossetie) [inline diagrams.net/assets/stencils/aws/deployment_management.xml:256]: This file is copied from https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/aws/deployment_management.xml#L256

You should probably fix this typo there

File: diagrams.net/assets/stencils/office/servers.xml [hunk 2]
```diff
@@ -1678,7 +1678,7 @@
         <fillstroke/>
     </foreground>
 </shape>
-<shape aspect="variable" h="59.17" name="Excahnge Client Access Server Role" strokewidth="inherit" w="51.75">
```
Comment (ggrossetie) [inline diagrams.net/assets/stencils/office/servers.xml:1681]: Same here: https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/office/servers.xml#L1681

File: server/src/main/resources/c4.puml [hunk 3]
```diff
@@ -730,7 +730,7 @@ $elementSkin
 ' Line breaks
 ' ##################################
 
-' PlantUML supports no DETERMINISTIC/automatic line breaks of "PlantUML line" (C4 Relashionships)
```
Comment (ggrossetie) [inline server/src/main/resources/c4.puml:733]: File copied from: https://github.com/plantuml-stdlib/C4-PlantUML/blob/0d97fb8713bfafbc6bb231e036d4a74388353c23/C4.puml#L872

File: diagrams.net/assets/stencils/aws/deployment_management.xml (hunk 1 above)
Comment (deining) [reply diagrams.net/assets/stencils/aws/deployment_management.xml:256]: > This file is copied from https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/aws/deployment_management.xml#L256
> 
> You should probably fix this typo there

Thanks for pointing that out. Meanwhile I authored this [pull request](https://github.com/jgraph/drawio/pull/3423) that addresses the issue in the upstream repository. I think we leave my change in the kroki repo in place.

File: diagrams.net/assets/stencils/office/servers.xml (hunk 2 above)
Comment (deining) [reply diagrams.net/assets/stencils/office/servers.xml:1681]: > Same here: https://github.com/jgraph/drawio/blob/bbe89f62e093c56735281e63ac42b3fccb07425d/src/main/webapp/stencils/office/servers.xml#L1681

My [pull request](https://github.com/jgraph/drawio/pull/3423) already mentioned above addresses this issue in the upstream repository, too.

File: server/src/main/resources/c4.puml (hunk 3 above)
Comment (deining) [reply server/src/main/resources/c4.puml:733]: > File copied from: https://github.com/plantuml-stdlib/C4-PlantUML/blob/0d97fb8713bfafbc6bb231e036d4a74388353c23/C4.puml#L872

Thanks for pointing that out. Meanwhile I authored this [pull request](https://github.com/plantuml-stdlib/C4-PlantUML/pull/292) that addresses the issue in the upstream repository. My PR was approved already 😄.

**Comment by ggrossetie** [comment]
All good then!
Thanks for your contribution.

**Pull request merged**